app/.jshintrc

```diff
@@ -1,6 +1,5 @@
 {
```

Comment:
replace jshint with eslint

Reply:
Turns out eslint was already here, so this file was not needed anymore, removed it :)